src/components/ShareStructure/ShareStructure.vue

```diff
@@ -614,9 +612,9 @@ export default class ShareStructure extends Mixins(ShareMixin) {
    * @params shareClass The Share class to compare
    */
   private isShareClassEdited (shareClass: ShareClassIF): boolean {
-    const originalShareClasses: any = this.isCorrection
+    const originalShareClasses = this.isCorrection
       ? cloneDeep(this.originalIA.incorporationApplication.shareStructure.shareClasses)
-      : cloneDeep(this.originalSnapshot[4].shareClasses)
```

Comment:
I could see no reason to case the type to `any`. In this case the type is ShareClassIF.

Same below.